core/src/main/java/com/scalar/db/api/UpdateBuilder.java

```diff
@@ -73,10 +76,14 @@ public Buildable partitionKey(Key partitionKey) {
   }
 
   public static class Buildable extends OperationBuilder.Buildable<Update>
-      implements ClusteringKey<Buildable>, Condition<Buildable>, Values<Buildable> {
+      implements ClusteringKey<Buildable>,
+          Condition<Buildable>,
+          Values<Buildable>,
+          Attribute<Buildable> {
     final Map<String, Column<?>> columns = new LinkedHashMap<>();
     @Nullable Key clusteringKey;
     @Nullable MutationCondition condition;
+    Map<String, String> attributes = new HashMap<>();
```

Comment:
Same as above.

Reply:
Fixed in 241b26e7c2cb06a6d89750265790236053857342.